### ISSUE TITLE
[Bug] Freecam only works if movement keybinds are not mouse buttons

### ISSUE THREAD

**Issue by LordVertice** [MeteorDevelopment/meteor-client#2722]
### Describe the bug

I use shift for sprint because why doesn't MC do that by default? Literally every game works that way. I crouch with my extra mouse button (MB4) because it's more convenient.

When in freecam, I can only go up with space, but not down at all with shift or MB4 (sprint and crouch, respectively).

It only lets you fly down if your crouch key is set to shift. The sprint key makes you sprint like expected, no matter what it's set to. For parity with spectator mode (and general functionality), freecam should move down with the crouch key.

The weird thing to me is that it'd make sense if it just ignored your crouch keybind and always went with the default, shift. But it's not using your keybind NOR the default, unless the keybind IS the default.

### Steps to reproduce

```bash
- Change your crouch key to anything but shift
- Enable freecam
- Try both shift and your new crouch key, freecam will not move downwards.

Sprinting always works. I did not try if going up works if you rebound space, but that doesn't seem probable anyways. You might want to check for it though.
```

### Meteor Version

0.5.1

### Minecraft Version

1.19.2

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by AnhNguyenlost13** [comment]
I still can move down fine with ctrl as crouch key, maybe try other standard keys?

**Comment by LordVertice** [comment]
Looks like I made a mistake during testing. The correct issue is: Freecam only works if movement keybinds are not mouse buttons.

I tried changing jump and crouch to random keyboard keys, that worked. But when I change them to MB4 and/or MB5 it no longer works. All other Meteor and Minecraft features can handle those keys just fine; for example flying in spectator or using boat fly. It's specifically freecam that can't handle mouse buttons. I might add that scrolling doesn't seem to work for me either, but I have a feeling that might be a separate issue.
